### PR TITLE
[CIAPP-6240] Clarifying documentation for CI Visibility Test Summaries

### PR DESCRIPTION
**Pull request by andrea-mosk**
### What does this PR do?

Clarifies documentation for CI Visibility Test Summaries

### Additional Notes

---

### Reviewer checklist
- [x] Review the changed files.
- [x] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [x] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/guides/pull_request_comments.md [hunk 1]
```diff
@@ -17,24 +17,26 @@ further_reading:
...
+debug any failed/flaky tests without leaving the pull requests view.
 
 {{< img src="ci/github_comments_light.png" alt="Datadog GitHub pull request comment preview" style="width:100%;">}}
 
 ### Compatibility
 
 This integration is only available for test services hosted on `github.com`.
 
-## Install the GitHub integration
+## Enable test summaries
```
Comment (drodriguezhdez) [inline content/en/continuous_integration/guides/pull_request_comments.md:32]:
```suggestion
## Enable test summaries in GitHub
```

File: content/en/continuous_integration/guides/pull_request_comments.md [hunk 2]
```diff
@@ -17,24 +17,26 @@ further_reading:
...
 
-To install the [GitHub integration][1]:
+Enabling test summaries is very quick and can be done in two simple steps. The first step is to install the [GitHub integration][1]:
```
Comment (drodriguezhdez) [inline content/en/continuous_integration/guides/pull_request_comments.md:34]:
```suggestion
You can enable test summaries in GitHub by following the next steps:

1. Install the [GitHub integration][1]:
```

File: content/en/continuous_integration/guides/pull_request_comments.md [hunk 3]
```diff
@@ -17,24 +17,26 @@ further_reading:
...
 
 1. Navigate to the **Configuration** tab on the [GitHub integration tile][2] and click **+ Create GitHub App**.
 2. Give the application read and write permissions for pull requests.
 
-## Enable pull request comments
-
-You can enable GitHub comments from the Test Service Settings page or the commit or branch page.
+The second step is to enable test summaries for a test service from the Test Service Settings page or from the commit/branch page.
```
Comment (drodriguezhdez) [inline content/en/continuous_integration/guides/pull_request_comments.md:39]:
```suggestion
2. Enable test summaries for a test service. You can do that either using the Test Service Settings page or from the commit/branch page.
```

File: content/en/continuous_integration/guides/pull_request_comments.md (hunk 2 above)
Comment (andrea-mosk) [reply content/en/continuous_integration/guides/pull_request_comments.md:34]: This is how I wrote that in principle, but then you have nested ordered lists. Something like:

1. Install Github integration:
    a. Navigate to the Configuration tab...
    b. Give the application...
    
2. Enable test summaries for a test service...

Which is something that I didn't really see in our UI in other places. Do you think that would be better?

Comment (drodriguezhdez) [reply content/en/continuous_integration/guides/pull_request_comments.md:34]: Yes, I'd say so IMO. I could locate an example for Jenkins (unordered though) https://github.com/DataDog/documentation/blob/master/content/en/continuous_integration/pipelines/jenkins.md?plain=1#L794-L805

Comment (andrea-mosk) [reply content/en/continuous_integration/guides/pull_request_comments.md]: 👍 Changed to be an ordered list!

**Review by drodriguezhdez — APPROVED** [review]

File: content/en/continuous_integration/guides/pull_request_comments.md [hunk 4]
```diff
@@ -17,26 +17,28 @@ further_reading:
...
 
-To install the [GitHub integration][1]:
+You can enable test summaries in pull requests with the following steps:
 
-1. Navigate to the **Configuration** tab on the [GitHub integration tile][2] and click **+ Create GitHub App**.
-2. Give the application read and write permissions for pull requests.
+1. Install the [GitHub integration][1]:
+   1. Navigate to the **Configuration** tab on the [GitHub integration tile][2] and click **+ Create GitHub App**.
+   2. Give the application read and write permissions for pull requests.
+2. Enable test summaries for one or more test services. It can be done from the Test Service Settings page or from the commit/branch page.
```
[the remark this answers is not in the document]
Comment (andrea-mosk) [reply content/en/continuous_integration/guides/pull_request_comments.md]: Changed!

File: content/en/continuous_integration/guides/pull_request_comments.md [hunk 5]
```diff
@@ -17,26 +17,28 @@ further_reading:
 <div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-Datadog integrates with GitHub to show test result summaries and error messages for failed tests in pull request comments.
+If you're using the Test Visibility product, Datadog can integrate with GitHub to show test results summaries directly
+in your pull requests. The summary contains an overview of the tests executions, flakiness information and
+error messages for failed tests.
+With this report, developers get instant feedback about the results of their tests, and they are able to
```
Comment (drichards-87) [inline content/en/continuous_integration/guides/pull_request_comments.md:23]:
```suggestion
With this report, developers get instant feedback about their tests results, including the ability to
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some minor feedback from Docs.

**Review by estherk15 — APPROVED** [review]

**Pull request merged**